packages/curve-ui-kit/src/themes/components/chip/mui-chip.ts

```diff
@@ -69,9 +69,9 @@ export const defineMuiChip = (
         '& .MuiChip-label': {
           paddingInlineStart: 0,
           paddingInlineEnd: Spacing.xs.desktop,
-          '&:empty': { display: 'none' },
         },
       },
+      '& .MuiChip-label:empty': { display: 'none' },
```

Comment:
This diff doesn't change anything functionally right? It's still the same selector as far as I can tell

Reply:
https://github.com/curvefi/curve-frontend/pull/1117#discussion_r2177229127